docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -145,7 +145,7 @@ For the most accurate migration assessment, the source database must be actively
 
     After generating the report, navigate to the **Migrations** tab in the YugabyteDB UI at <http://127.0.0.1:15433> to see the available migrations.
 
-    ![Migration Landing Page](/images/migrate/ybd-landing-page.png)
+    ![Migration Landing Page](/images/migrate/migration-list-page.png)
```

Comment:
Is this the Migrations tab? Did the screenshot crop out the black bar that would tell me where I am?